### DISCUSSION TITLE
关闭了插件市场的jwt-Auth，访问服务时仍然进行Jwt验证

### DISCUSSION THREAD

**Discussion by olivercp3** [alibaba/higress#474] [Q&A]
你好，开启jwt-Auth之后能够正常拦截，关闭之后仍被拦截。

> curl xxx.api.com
Jwt is missing

由于我的集群节点非常多，ingress也很多，每次higress-gateway 都要经过很长时间才能READY。

是否有此因素导致，jwt关闭后仍然在验证jwt呢，还是缓解等，以上都是我的猜测。

**Comment by johnlanni** [comment]
https://github.com/alibaba/higress/discussions/475

跟这个是一个问题，k8s service太多，可能导致推送阻塞，可以先把这个配置开启

**Reply by johnlanni** [reply]
可以先重启下数据面，看看如果jwt正确关闭了，那一定就是这个问题

**Reply by johnlanni** [reply]
如果担心这个配置导致的变更窗口问题，可以先给higress-gateway配置这个环境变量
PILOT_XDS_SEND_TIMEOUT: "60s"

可以触发xds推送阻塞后的超时重试
